src/Microsoft.CodeAnalysis.Analyzers/Core/CodeAnalysisDiagnosticsResources.resx

```diff
@@ -529,4 +529,13 @@
   <data name="PreferIsKindFix" xml:space="preserve">
     <value>Use 'IsKind' instead of 'Kind'</value>
   </data>
+  <data name="MissingSourceGeneratorAttributeDescription" xml:space="preserve">
+    <value>'{0}' inherits from ISourceGenerator but does not have the [Generator] attribute</value>
```

Comment:
Enclose `ISourceGenerator` between single quotes, and possibly use "implements" instead of "inherits". Inheritance is used more with classes.

Reply:
Probably worth to add single quotes also around `[Generator]`.